src/components/GroupList.tsx

```diff
@@ -1,6 +1,103 @@
-
-import { Flex } from '@chakra-ui/react';
-
+import {
+  Container,
+  Box,
+  Flex,
+  useTheme,
+  Avatar,
+  Heading,
+  Text,
+  Stack,
+  StackDivider,
+  Image,
+} from "@chakra-ui/react";
+import netflixImage from "./images/netflix.png";
```

Comment:
This should be `../images/netflix.png` since it is now in the components folder which is one directory down. Ditto for below.

Reply:
fixed